recipe/0068-newspaper/index.md

```diff
@@ -1,43 +1,253 @@
 
-## Restrictions
+### Date of publication or navDate
+One of the important features of Newspapers is the publication date which can allow the user to navigate issues by the date they were published. To achieve this we use the special [navDate](https://iiif.io/api/presentation/3.0/#navdate) property. This `navDate` should be added both to the items in a collection and also in the issue manifest. This allows viewers to present a date based navigation for Newspaper collections. 
 
-When is this pattern is usable / not usable? Is it deprecated? If it uses multiple specifications, which versions are needed, etc.? (Not present if not needed.)
+The `navDate` is added to the collection in the items property for example:
 
-## Example
+Full JSON: [newspaper_title-collection.json](newspaper_title-collection.json)
+
+{: class="line-numbers line-highlight" data-line="10,20"}
+```json
+"items": [
+    {
+      "id": "https://../newspaper_issue_1-manifest.json",
+      "type": "Manifest",
+      "label": {
+        "de": [
+          "Berliner Tageblatt - 1925-02-16"
+        ]
+      },
+      "navDate": "1925-02-16T00:00:00Z"
+    },
+    {
+      "id": "https://../newspaper_issue_2-manifest.json",
+      "type": "Manifest",
+      "label": {
+        "de": [
+          "Berliner Tageblatt - 1925-03-13"
+        ]
+      },
+      "navDate": "1925-03-13T00:00:00Z"
+    }
+]
+```
+
+The `navDate` should also be present in the issue Manifest and can be seen in the example below:
+
+Full JSON: [newspaper_issue_1-manifest.json](newspaper_issue_1-manifest.json)
+
+{: class="line-numbers line-highlight" style="max-height: 15em;" data-line="9"}
+```json
+{
+    "id": "https://.../newspaper_issue_1-manifest.json",
+    "type": "Manifest",
+    "label": {
+            "de": [
+                    "Berliner Tageblatt - 1925-02-16"
+            ]
+    },
+    "navDate": "1925-02-16T00:00:00Z"
+}, 
+```
 
-Describe in prose and provide examples, e.g.: 
+For Editions, a temporal value can be inserted to enforce navigation order. `navDate` is not an assertion of when an issue was published but instead a datetime useful for navigation. Therefore, you can use a `06:00` timestamp for a morning edition and a `17:00` for an evening edition to provide browse order.
 
-``` json-doc
+### Linking to Annotations
+Digitised Newspapers often have associated OCR text and to make this available in a IIIF viewer it needs to be in the form of W3C annotations linked from the Newspaper page. An example of this link between page (or IIIF canvas) can be seen highlighted below:
+
+Full JSON: [newspaper_issue_1-manifest.json](newspaper_issue_1-manifest.json)
+
+{: class="line-numbers line-highlight" data-line="35-40"}
+```json
 {
-  "@context": [
-    "http://www.w3.org/ns/anno.jsonld",
-    "http://iiif.io/api/presentation/{{ page.major }}/context.json"
+  "id": "https://iiif.europeana.eu/presentation/9200355/BibliographicResource_3000096302513/canvas/p1",
```

Comment:
The `id` should be a IIIF recipe URI. 
And this example should be called in from the JSON file, not rewritten with different URIs.

Reply:
Should this be? Note the Europeana Canvas Ids are also used in the linked full JSON. The `Id` of the manifest, annotation pages and collection is a recipe resolvable URL  but I haven't changed the canvas ids... Should we?